doc/api/errors.md

```diff
@@ -644,6 +644,17 @@ according to the encoding provided.
 Used by the `util.TextDecoder()` API when the encoding provided is not one of
 the [WHATWG Supported Encodings][].
 
+<a id="ERR_EVENTS_MAX_LISTENERS_TYPE"></a>
+### ERR_EVENTS_MAX_LISTENERS_TYPE
```

Comment:
This type is not used anymore and should be removed.

Reply:
oh right... heh, forgot about that ;-)